src/Kernel/IQSharpEngine.cs

```diff
@@ -101,7 +101,6 @@ private void RegisterPackageLoadedEvent(IServiceProvider services, ILogger logge
                 // Except assemblies known at compile-time as well.
                 .Add(typeof(StateVectorToHtmlResultEncoder).Assembly.GetName())
                 .Add(typeof(AzureClientErrorToHtmlEncoder).Assembly.GetName());
-            foreach (var knownAssembly in knownAssemblies) System.Console.WriteLine($"{knownAssembly.FullName}");
```

Comment:
Instead of removing, could we do a `logger.Info` or `logger.Debug`?

Reply:
Yes, I don't think this is very useful except for debugging, so using `logger.Debug` makes sense to me.